bases/br_ms_sim/code/sub/agregacao.do

```diff
@@ -102,10 +102,35 @@ foreach ano of numlist 1996(1)2018 {
 			
 			order id_municipio
 			
-			export delimited "output/municipio/ano=`ano'/estado_abrev=`estado_abrev'/municipio.csv", replace
+			export delimited "output/municipio/ano=`ano'/sigla_uf=`sigla_uf'/municipio.csv", replace datafmt
 			
 		restore
 		
 	}
 }
 *
+
+
+/*
+gen causa = substr(causa_basica, 1, 3)
+
+local cat_overdose	F10 F11 F12 F14 F16 F19 ///
+					T40 T41 T42 T43 T44 T45 T46 T47 T48 T49 T50 ///
+					X42 X43 X44 X45 X46 X47 X48 X49 ///
+					X60 X61 X62 X63 X64 X65 X66 X67 X68 X69 ///
+					Y12 Y13 Y14 Y15 Y16 Y49 Y50 Y51 ///
+					Z64 Z65
+
+local cat_violento	W33 W34 ///
+					X85 X86 X87 X88 X89 X90 X91 X92 X93 X94 X95 X96 X97 X98 X99 ///
+					Y00 Y01 Y02 Y03 Y04 Y05 Y06 Y07 Y08 Y09 Y10 Y11 Y12 Y13 Y14 Y15 Y16 Y17 Y18 Y19 Y20 Y21 Y22 Y23 Y24 Y25 Y26 Y27 Y28 Y29 Y30 Y31 Y32 Y33 Y34 Y35 Y87 Y89
+
+foreach g in violento overdose {
```

Comment:
eu não entendi esse loop pois as variáveis referidas não são as mesmas mas posso só estar desatualizado em Stata e estar viajando, se for o caso perdão!!